### PR TITLE
add qna

### PR DESCRIPTION
**Pull request by shivamsoni00**

### CODE REVIEW

File: Components/questionAns/Q0/q0.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            <p class="option" onclick="changes()">
+                Option D
+            </p>
+        </div>
+        <div class="q-btn">
+            <a href="../../category/category.html" class="btn outline-secondary ques-btn0">Back</a>
+            <a href="../Q1/q1.html" class="btn outline-secondary ques-btn1">Next</a>
+        </div>
+    </div>
+    <script type="text/javascript">
```
Comment (sah-shah7) [inline Components/questionAns/Q0/q0.html:54]: Can use a separate JS file and link it. Rest looks okay

File: Components/questionAns/Q0/q0.css [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+	display: flex;
+	flex-direction: column;
+	gap: 1rem;
+	justify-content: center;
+	align-items: center;
+	justify-items: center;
+	width: 100vw;
+	height: 100vh;
+	overflow-x: hidden;
+    background-color: var(--default-color);
```
Comment (sah-shah7) [inline Components/questionAns/Q0/q0.css:13]: Use code Formatter

**Review by sah-shah7 — COMMENTED** [review]
Rest Looks okay.

File: Components/questionAns/Q1/q1.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            <p class="option" onclick="changes()">
+                Option D
+            </p>
+        </div>
+        <div class="q-btn">
+            <a href="../Q0/q0.html" class="btn outline-secondary ques-btn0">Back</a>
+            <a href="../Q2/q2.html" class="btn outline-secondary ques-btn1">Next</a>
+        </div>
+    </div>
+    <script type="text/javascript">
```
Comment (AurobindoGupta) [inline Components/questionAns/Q1/q1.html:54]: u can put this in a separate js file

File: Components/questionAns/Q2/q2.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+            <p class="option" onclick="changes()">
+                Option D
+            </p>
+        </div>
+        <div class="q-btn">
+            <a href="../Q1/q1.html" class="btn outline-secondary ques-btn0">Back</a>
+            <a href="../Q3/q3.html" class="btn outline-secondary ques-btn1">Next</a>
+        </div>
+    </div>
+    <script type="text/javascript">
```
Comment (AurobindoGupta) [inline Components/questionAns/Q2/q2.html:54]: same

**Review by AurobindoGupta — APPROVED** [review]
nice work !!

**Pull request merged**